src/filters/filters.less

```diff
@@ -28,3 +28,6 @@
     }
   }
 }
+.category-select{
+  display: flex;
+}
```

Comment:
Prefix this with filter-pf or child it to the filter-pf selector.

Not due to this PR, but I see in the same file there is .input-group... could you child that to the filter-pf while you are in there?